src/draw/DefaultDrawer.js

```diff
@@ -350,13 +350,18 @@ class DefaultDrawer {
     const origParent = this.pluginData.getComponentById(event.subject.parent.data.id);
     const target = dropTarget ? d3.select(dropTarget) : null;
 
-    if (target === origParent) {
-      const { x, y } = event.subject;
+    if (target === origParent
+      || (target && target.datum().data && origParent
+        && origParent.id === target.datum().data.id)) {
+      const { x, y } = event/* .subject */;
```

Comment:
Missing to remove this comment or have to reword on it.

Reply:
removed